ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -657,6 +666,14 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
                 << errno
                 << "(" << strerror(errno) << ")"
                 << endl;
+            const char* errorMessage = ss.str().c_str();
```

Comment:
I'm not convinced this call sequence is safe. As far as I'm aware `stringstream::str()` returns a temporary `string` object, so the string buffer returned by `c_str()` on that temp object will presumably also be transient. The solution would be to make `errorMessage` a `string` storing the result of `ss.str()` so that doesn't go out of scope until `s_callbacks.LogError(errorMessage.c_str())` returns.

Reply:
@pmj let me know if the update looks good